src/pages/api/aws/sign-s3.ts

```diff
@@ -0,0 +1,49 @@
+import {NextApiRequest, NextApiResponse} from 'next'
+import AWS from 'aws-sdk'
```

Comment:
this is the v2 of the aws sdk because the modern v3 "modular" sdk documentation made me very sad and I couldn't get this flow figured out